modules/ROOT/pages/_common-content/productized-instructions.adoc

```diff
@@ -0,0 +1,61 @@
+:page-partial:
+
+:rhbq_config_registry_url: https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-2-7/guide/f93c45bd-4feb-4f74-a70a-022e9fb41957#_9064727c-4b8a-4068-a9ba-8de6b258a14a
+:rhbq_config_maven_url: https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-2-7/guide/f93c45bd-4feb-4f74-a70a-022e9fb41957#proc_online-maven_quarkus-getting-started
+:redhat_registry_auth_url: https://access.redhat.com/RegistryAuthentication
+
+[[proc-configuring-maven-rhbq]]
+== Configuring your Maven project to Red Hat build of Quarkus and OpenShift Serverless Logic
+
+To use the Red Hat Build of Quarkus (RHBQ) libraries, you need to configure your local Maven installation.
+
+.Procedure
+. To configure the Apache Maven `settings.xml` file and Quarkus extension registry client, follow the instructions in the link:{rhbq_config_maven_url}[Configuring the Maven settings.xml file for the online repository] and link:{rhbq_config_registry_url}[Configuring Quarkus extension registry client].
```

Comment:
One thing we are missing here is the change of the groupId from `io.quarkus.platform` to `com.redhat.quarkus.platform` and change of the version. Otherwise, the project will still use the community Quarkus version (together with the Maven plugin). Luckily, we can just reference [this part of the RHBQ docs](https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-2-7/guide/f93c45bd-4feb-4f74-a70a-022e9fb41957#proc_reconfiguring-maven-community-dependencies_quarkus-getting-started) as the last step.